### ISSUE TITLE
hardcoded alt="Nature Image 1" in Featured Image (Admin Page)

### ISSUE THREAD

**Issue by khairendra** [owen2345/camaleon-cms#97]
In edit or create post, the Featured image have img alt="Nature Image 1" it should be remove or better have an option to set the alt text.

**Comment by owen2345** [comment]
fixed!
